### ISSUE TITLE
`php_fpm` possible conflict. 

### ISSUE THREAD

**Issue by postadelmaga** [toy/image_optim#76]
I have integrated this tool with a web app. based on php.
I have tested it on my local env, based on `mod_php` and I have no issue but when I deploy the stuff to live server `image_optim` is not compressing the images.
The live server uses `php_fpm` and I suspect this is the reason because of the issue ...

Do you have any idea how can I debug it ?
I'm executing `image_optim` using php `exec()` ... the strange thing is I got not error, the output show 0 optimization that's it. But If I try to optimize the same image in my local environment the image get reduced.

**Comment by toy** [comment]
You can check output of `image_optim --info`, compare with output of same command in local environment, don't miss stderr.
Also you can add `--verbose` to optimization command and also analyse output.

**Comment by toy** [comment]
@postadelmaga Do you have any input on this?

**Comment by postadelmaga** [comment]
`image_optim --info` didn't show any difference, I had to drop the use of `image_optim` since it was not compatible with the server environment and I'm not managing the server.

**Comment by toy** [comment]
I am sorry to hear that. Please reopen the issue if you will reconsider and would like to investigate the issue.

**Comment by HansVanEijsden** [comment]
I have the same problem and would like to open this issue.

I use image_optim v0.26.0 and all goes well from the shell. From PHP it also works, but it barely optimizes the images and it also doesn't find svgo and pngout when executed from php, although I have installed them and they can be found from the shell (also from the same user as the php user): that's the visible difference in the --info output.
I tried using a simple wrapper bash script: no luck, still the missing svgo and pngout binaries when executed through php.

PHP 7.1.12, Linux 4.13.0-0.bpo.1-amd64 #1 SMP Debian 4.13.13-1~bpo9+1 (2017-11-22) x86_64 GNU/Linux
Just ask if you want to know more.
